views/templates/front/adyencheckout.tpl

```diff
@@ -0,0 +1,19 @@
+{if !$originKey}
+    <h5>There is an error with retrieving the originKey,
+        please check your API key in the Adyen Module configuration</h5>
+{else}
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+    <script>
+        $(document).ready(function () {
+            window.adyenCheckout = new AdyenCheckout({
```

Comment:
can we also pass here the paymentMethods response so we don't have to do it everyime for the other components?

Reply:
it is on another template at the moment, will find a way to not duplicate paymentMethods response